scopesim/effects/ter_curves.py

```diff
@@ -213,6 +221,8 @@ def plot(self, which="x", wavelength=None, *, axes=None, **kwargs):
         abbrs = {"t": "transmission", "e": "emission",
                  "r": "reflection", "x": "throughput"}
 
+        if not isinstance(axes, list):
```

Comment:
AFAIK, matplotlib puts axes in an array, not a list. Also, what if someone passes a `tuple`? Maybe check against `collections.abc.Iterable` instead...